python_cross_site_scripting/GitsecureHQ_gitsecure-demo/mark_safe.py

```
@@ -0,0 +1,40 @@
+from django.http import HttpResponse
+from django.template import loader
+
+def not_really_safe(request):
+    template = loader.get_template('contents.html')
+    not_actually_safe = mark_safe(
+        """
+        <div>
+            <p>Content! %s</p>
+        </div>
+        """ % request.POST.get("contents")
+    )
```

Ruff (0.13.3)

Use of `mark_safe` may expose cross-site scripting vulnerabilities

(S308)